tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3611,4 +3611,37 @@ sdg/SG_GEN_PARL.SEX--F,proportion of seats held by women in national parliaments
 Count_Person_Upto18Years,children population count
 Count_Person_18OrMoreYears,adult population count
 Count_Person_65OrMoreYears,senior population count
-dc/x6l3mg60hpe7c,"Consumer Price Index of Consumer Goods And Services (Seasonally Unadjusted): Urban Consumer; Consumer Price Index for All Urban Consumers"
\ No newline at end of file
+dc/x6l3mg60hpe7c,"Consumer Price Index of Consumer Goods And Services (Seasonally Unadjusted): Urban Consumer; Consumer Price Index for All Urban Consumers"
+Count_Person_15OrMoreYears_EducationSector_Employed,Number of people employed in education sector
+Count_Person_15OrMoreYears_Manufacturing_Employed,Number of people employed in Manufacturing
+Count_Person_15OrMoreYears_SpecializedScientificAndTechnicalActivities_Employed,Number of people employed in specialized scientific or technical activities
+Count_Person_15OrMoreYears_AgricultureHuntingForestryCuttingTrees_Employed,Number of people employed in agriculture, forestry or fishing
+Count_Person_15OrMoreYears_HumanHealthAndSocialWorkActivities_Employed,Number of people employed in social work activities
+Count_Person_15OrMoreYears_PublicAdministrationAndDefence_Employed,Number of people employed in public administration or defence
+Count_Person_15OrMoreYears_InformationAndTelecommunication_Employed,Number of people employed in Information and communication
+Count_MortalityEvent_Infant_Rural,Rural infant mortality rate
+Count_MortalityEvent_Infant_Urban,Urban infant mortality rate
+Count_MedicalConditionIncident_ConditionMalaria,Number of malaria cases
+Count_MortalityEvent_Malaria_AsAFractionOf_Count_Death,Malaria mortality rate
+Count_HealthEstablishment_PublicAndCentralHospital,Number of public and central hospitals
+Count_CriminalIncidents,Number of criminal activities
+Count_Establishment_SocialCareFacility,Number of social care facilities
+Percent_Student_AsAFractionOf_Count_Teacher,Student teacher ratio
+Count_Person_Agriculture_Employed,Number of people employed in agriculture sector
+Count_Person_ConstructionSector_Employed,Number of people employed in construction sector
+Count_Person_ManufacturingSector_Employed,Number of people employed in manufacturing sector
+Count_Person_TradeSector_Employed,Number of people employed in trade sector
+Count_Person_SlumArea,Number of people living in slums
+UnemploymentRate_Person_15To65Years,Unemployment rate
+Count_MortalityEvent_Newborn_AsAFractionOf_Count_BirthEvent_LiveBirth,Newborn mortality rate
+Count_MortalityEvent_Infant_AsAFractionOf_Count_BirthEvent_LiveBirth,Infant mortality rate
+Count_MaternalMortalityEvent,Number of maternal mortality

```

Comment:
`Number of maternal deaths` to fix grammar.

Reply:
Thanks Ajai for sending the correction. Will be more careful from now on.